pyramid_swagger/api.py

```diff
@@ -268,14 +268,20 @@ def _get_target_url(spec, target, current_path=''):
 
```

Comment:
Let's add an entry to https://github.com/striglia/pyramid_swagger/blob/master/docs/changelog.rst under the "development version" describing what you fixed and a link to the issue for future readers.

I can release 2.4.0 after we merge this in.